src/h5web/visualizations/heatmap/ColorMapSelector.tsx

```diff
@@ -1,137 +1,83 @@
-import React, { CSSProperties } from 'react';
-import Select, {
-  components,
-  OptionProps,
-  IndicatorProps,
-  SingleValueProps,
-} from 'react-select';
-import { FiChevronDown } from 'react-icons/fi';
-import shallow from 'zustand/shallow';
-import { generateCSSLinearGradient, convertToOptions } from './utils';
-import {
-  INTERPOLATORS,
-  MOST_USED,
-  SINGLE_HUE,
-  MULTI_HUE,
-  CYCLICAL,
-  DIVERGING,
-} from './interpolators';
-import { useHeatmapConfig } from './config';
-import styles from './HeatmapToolbar.module.css';
-import { ColorMapOption } from './models';
-import { customThemeForSelect } from '../shared/utils';
+import React, { ReactElement } from 'react';
+import { Wrapper, Button, Menu, MenuItem } from 'react-aria-menubutton';
 
-const colorMapOptions = [
-  {
-    label: 'Common',
-    options: convertToOptions(MOST_USED),
-  },
-  {
-    label: 'Single hue',
-    options: convertToOptions(SINGLE_HUE),
-  },
-  {
-    label: 'Multi hue',
-    options: convertToOptions(MULTI_HUE),
-  },
-  {
-    label: 'Cyclical',
-    options: convertToOptions(CYCLICAL),
-  },
-  {
-    label: 'Diverging',
-    options: convertToOptions(DIVERGING),
-  },
-];
```

Comment:
No need to pre-process the color maps with `convertToOptions` since we no longer need to pass an `options` object -- we just render the options (`MenuItem`) ourselves instead.